tests/integration/api/test_eda_credential.py

```diff
@@ -134,6 +134,12 @@ def test_list_eda_credentials_with_kind_filter(
     )
     assert len(response.data["results"]) == 2
 
+    response = client.get(
+        f"{api_url_v1}/eda-credentials/?credential_type__kind=scm"
+        "&credential_type__kind=registry&name=m",
```

Comment:
@hsong-rh instead of m can we use a name_prefix and set it to the first character of the credential name